alcs-frontend/src/app/features/notice-of-intent/decision/decision-v2/decision-input/decision-conditions/decision-condition/decision-condition.component.ts

```diff
@@ -83,10 +82,6 @@ export class DecisionConditionComponent implements OnInit, OnChanges {
       description: this.data.description ?? null,
     });
 
-    if (this.showSingleDateField && this.dates.length > 0 && this.dates[0].date) {
```

Comment:
Why was this removed?

Reply:
Same as above:
Because singleDate field was removed from ApplicationDecisionConditionDto and replaced by dates.